.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
   - "10"
+  - "12"
+  - "14"
```

Comment:
```suggestion
  - "lts/*"
  - "node"
```

v10 isn't even supported anymore.

Reply:
Good catch! That makes it a lot easier 👍